content/news/2020-09-28-wallpapers-contest.md

```diff
@@ -0,0 +1,34 @@
++++
+type = "news"
+title = "Call for Wallpapers"
+author = "bitigchi"
+date = "2020-10-01 13:00:00+03:00"
+tags = ["wallpapers", "contest"]
++++
+
+One of the most common feedback that we receive is that the Haiku needs some nice colourful wallpapers shipping with Haiku releases. Suffice to say, that we agree with this completely. We definitely want to improve the current situation. And we need your help.
+
+Haiku currently ships with only one (1) wallpaper, and it's not a complete wallpaper per se, it's just the Haiku logo positioned on the screen on an appropriate position according to the screen resolution. To be fair, there is a reason we only use the Haiku logo as the default wallpaper; we want Haiku to be usable on systems that have memory as low as 256 MB, therefore each byte that we can spare counts.
+
+We are looking for artists that can contribute to Haiku with some appealing wallpapers. To be more precise, we are looking for either:
+
+- Vector illustrations
+- Painted illustrations
+
+## Guidelines
+
+Artists are free to submit in any category they please. We hope below guidelines will help you in creating the new Haiku wallpapers:
+
+- Illustrations should be in at least landscape 5K size.
+- Vector illustrations should include a "lite" version, to be used as the default wallpaper (see the second paragraph). Color scheme of this lite variant should blend in with the standard system background colour.
+- We encourage painted illustrations to use subtle haiku themes, such as leaves, Far-Eastern typography, and literature. Anime-themed drawings are also welcome, but should there be human characters, they should not be the main focus.
+- For vector illustrations, use darker blue tones as the main colour. For painted illustrations, use warmer colours.
```

Comment:
I don't see why a vector graphic shouldn't have warm colours, or bitmaps have dark blue ones. It's weirdly specific... :)

Reply:
I just thought we would like to keep the blue tones that we use today. For drawings, it should be easy on the eyes. Moreover, all the Far-Eastern drawings I've encountered had warmer colours and natural elements.

Reply:
I just don't see the distinction made between vector and bitmap. I do see that a logo that's supposed to overlay the standard blue background has to fit in, ofc.

Reply:
They are two different categories. Bitmap can be anything, but the vector is supposed to be a new default background, expanding on the current theme. That's why we need a vector lite version.

Or we can remove all constraints, let artists do what they please, and implement a file selector according to the RAM and resolution.